src/Components/ImagesTable/Target.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  useGetClonesQuery,
+  useGetComposeStatusQuery,
+} from '../../store/apiSlice';
+
+type TargetProps = {
+  composeId: string;
+};
+
+const targetOptions = {
+  aws: 'Amazon Web Services',
+  'aws.s3': 'Amazon Web Services',
+  azure: 'Microsoft Azure',
+  gcp: 'Google Cloud Platform',
+  vsphere: 'VMWare',
+  'guest-image': 'Virtualization - Guest image',
+  'image-installer': 'Bare metal - Installer',
+  'edge-commit': 'edge-commit',
+  'edge-installer': 'edge-installer',
+  ami: 'ami',
+  'rhel-edge-commit': 'rhel-edge-commit',
+  'rhel-edge-installer': 'rhel-edge-installer',
+  vhd: 'vhd',
+};
+
+const Target = ({ composeId }: TargetProps) => {
+  const { data: clones, isSuccess: isSuccessClones } =
+    useGetClonesQuery(composeId);
+  const { data: composeStatus, isSuccess: isSuccessTarget } =
+    useGetComposeStatusQuery(composeId);
+  const target = composeStatus?.request?.image_requests[0];
```

Comment:
I think this variable name is confusing, maybe call it imageReq?

Reply:
+1

It would also allow to change the `tar` back to `target` which I'd say would be preferable.

Reply:
done @regexowl thank you